test/spec/spectest.cpp

```diff
@@ -612,44 +628,45 @@ void SpecTest::run(std::string_view Proposal, std::string_view UnitName) {
         return;
       }
       case CommandID::AssertTrap: {
-        const auto &Action = Cmd["action"s];
-        const auto &Text = Cmd["text"s].Get<std::string>();
-        const uint64_t LineNumber = Cmd["line"].Get<uint64_t>();
-        TrapInvoke(Action, Text, LineNumber);
+        const simdjson::dom::object Action = Cmd["action"].get_object();
```

Comment:
Use reference instead of copying a new data structure.

Reply:
Hey @hydai , I tried using reference here, but unfortunately it was causing SegFault in the spectests

Reply:
I have added the references in https://github.com/WasmEdge/WasmEdge/pull/2659/commits/4e71119fc68455dceb5b27fc933a93c451f63ecc, it was failing due to `get_object()` function calls.